test/micro/org/openjdk/bench/java/text/DefFormatterBench.java

```diff
@@ -69,6 +92,21 @@ public void testDefNumberFormatter(final Blackhole blackhole) {
         }
     }
 
+    @Benchmark
+    public void testDateFormat(final Blackhole blackhole) {
+        blackhole.consume(dateFormat.format(date));
```

Comment:
We usually just return the value if the blackhole only consumes one value. Blackhole is usually used for producing multiple values, such as running in a loop and ensure each iteration's result is used (as in testDefNumberFormatter)